### ISSUE TITLE
ClientKafka and KafkaServer naming conventions become much simpler

### ISSUE THREAD

**Issue by smhmayboudi** [nestjs/nest#5603]
## Is your feature request related to a problem? Please describe.

"Kafka client and consumer naming conventions can be customized by extending ClientKafka and KafkaServer in your own custom provider and overriding the constructor."

Customizing a name becomes very complex since it can be solved with Kafka options.

## Describe the solution you'd like

Adding two string to Kafka options and use them in ServerKafka and ClientKafka. Since those values are optional, there is no drawback but it saves a lot of code and effort.

## Teachability, Documentation, Adoption, Migration Strategy

https://github.com/nestjs/nest/blob/master/packages/microservices/interfaces/microservice-configuration.interface.ts
```ts
export interface KafkaOptions {
  transport?: Transport.KAFKA;
  options?: {
     clientIdPostfix?: string;
     serverIdPostfix?: string;
     ...
  }
}
```

https://github.com/nestjs/nest/blob/master/packages/microservices/client/client-kafka.ts
```ts
    this.clientId =
      (clientOptions.clientId || KAFKA_DEFAULT_CLIENT) + (clientOptions.clientIdPostfix || '-client');
    this.groupId = (consumerOptions.groupId || KAFKA_DEFAULT_GROUP) + (clientOptions.clientIdPostfix || '-client');
```

https://github.com/nestjs/nest/blob/master/packages/microservices/server/server-kafka.ts
```ts
    this.clientId =
      (clientOptions.clientId || KAFKA_DEFAULT_CLIENT) + (clientOptions.serverIdPostfix || '-server');
    this.groupId = (consumerOptions.groupId || KAFKA_DEFAULT_GROUP) + (clientOptions.serverIdPostfix || '-server');
```

## What is the motivation / use case for changing the behavior?

I am using Kafka as my backbone of event-driven microservice and I use '_' instead of '-' in the patterns.

**Comment by kamilmysliwiec** [comment]
Would you like to create a PR for this issue?

**Comment by smhmayboudi** [comment]
Yes. Would you please provide me a guideline?

**Comment by smhmayboudi** [comment]
I find out, I can do it just with one variable as clinetId.

More importantly, why Kafka interface is copied in two places.
`nest/packages/common/interfaces/external/kafka.interface.ts`
`nest/packages/microservices/external/kafka.interface.ts`

**Comment by kamilmysliwiec** [comment]
Fixed in 7.5.0

**Comment by sgentile** [comment]
I'm specifying a group-id and it's appending client and server to it - they need to be the same so why is the library appending values that I didn't specify ?
